src/psyclone/dynamo0p3.py

```diff
@@ -59,24 +59,45 @@
     FORTRAN_INTENT_NAMES, DataAccess, Literal, Reference, Schedule, \
     CodedKern, Schedule
 
-# First section : Parser specialisations and classes
-
-# Function spaces (FS)
+# --------------------------------------------------------------------------- #
```

Comment:
My intention here was to make "headers" for different functionality sections and subsections as the file is very long and it can be difficult to find a specific thing. Ideally, all these headers would not be required as we would have different pieces of LFRic API functionality in more than one file (see Joerg's restructuring quesion #474).  I googled for a standard way of commenting in Python and could not find anything special so I guess this is as good way as any - I am open to suggestions how to organise and mark up comments in code!